app/models/resource_action_workflow.rb

```diff
@@ -23,30 +23,44 @@ def dialogs
   end
 
   def submit_request
-    result = {}
+    process_request('ordered')
```

Comment:
We should update the service order text value to use the constants defined in the ServiceOrder model.
For example: `process_request(ServiceOrder::STATE_ORDERED)`